content/pt/docs/concepts/sampling/index.md

```diff
@@ -0,0 +1,215 @@
+de amostragem o mais cedo possível. A decisão de amostrar ou descartar um trecho
+ou um rastro não é feita inspecionando o rastro como um todo.
+
+Por exemplo, a forma mais comum de amostragem de cabeça é a
+[Amostragem de Probabilidade Consistente](/docs/specs/otel/trace/tracestate-probability-sampling/#consistent-probability-sampling).
+Isso também é conhecido como Amostragem Determinística. Neste caso, uma decisão
+de amostragem é tomada com base no ID do rastreio e na porcentagem desejada de
+rastreios a serem amostrados. Isso garante que rastreios inteiros sejam
+amostrados - sem trechos faltantes - a uma taxa consistente, como 5% de todos os
+rastreios.
+
+As vantagens da amostragem de cabeça são:
+
+- Fácil de entender
+- Fácil de configurar
+- Eficiente
+- Pode ser feita em qualquer ponto do _pipeline_ de coleta de rastreios
+
+A principal desvantagem da amostragem de cabeça é que não é possível tomar uma
+decisão de amostragem com base nos dados do rastreio inteiro. Por exemplo, você
+não pode garantir que todos os rastreios com um erro sejam amostrados apenas com
+a amostragem de cabeça. Para essa situação e muitas outras, você precisa da
+amostragem de cauda.
+
+## Amostragem de Cauda
+
+A amostragem de cauda é onde a decisão de amostrar um rastreio acontece
+considerando todos ou a maioria dos trechos dentro do rastreio. A Amostragem de
+Cauda oferece a opção de amostrar seus rastreios com base em critérios
+específicos derivados de diferentes partes de um rastreio, o que não é uma opção
+com a Amostragem de Cabeça.
+
+![Ilustração mostrando como os trechos se originam de um trecho raiz. Após os trechos serem concluídos, o processador de amostragem de cauda toma uma decisão de amostragem.](tail-sampling-process.png)
+
+Alguns exemplos de como você pode usar a Amostragem de Cauda incluem:
+
+- Sempre amostrar rastreios que contenham um erro
+- Amostrar rastreios com base na latência geral
+- Amostrar rastreios com base na presença ou valor de atributos específicos em
+  um ou mais trechos de um rastreio; por exemplo, amostrar mais rastreios
+  originados de um serviço recém-implantado
+- Aplicar diferentes taxas de amostragem a rastreios com base em certos
+  critérios, como quando os rastreios vêm apenas de serviços de baixo volume
+  versus rastreios com serviços de alto volume.
+
+Como você pode ver, a amostragem de cauda permite um grau muito maior de
+sofisticação na forma como você amostra os dados. Para sistemas maiores que
+devem amostrar telemetria, é quase sempre necessário usar a Amostragem de Cauda
+para equilibrar o volume de dados com a utilidade desses dados.
+
+Existem três principais desvantagens na amostragem de cauda atualmente:
+
+1. A amostragem de cauda pode ser difícil de implementar. Dependendo dos tipos
+   de técnicas de amostragem disponíveis para você, nem sempre é algo do tipo
+   "configurar e esquecer". À medida que seus sistemas mudam, suas estratégias
```

Comment:
```suggestion
1. A amostragem de cauda pode ser difícil de implementar. Dependendo dos tipos
   de técnicas de amostragem a sua disposição, nem sempre é uma decisão do tipo
   "configurar e esquecer". À medida que seus sistemas mudam, suas estratégias
```

Eu acho que ficou um pouco ambíguo em português, então tentando deixar mais claro o que o "it" significa.